src/base/memory_monitor_sr.cpp

```diff
@@ -169,6 +169,62 @@ namespace cubmem
 	  }
       }
   }
+
+  void memory_monitor::aggregate_server_info (MMON_SERVER_INFO &server_info)
+  {
+    strncpy (server_info.server_name, m_server_name.c_str (), m_server_name.size () + 1);
+    server_info.total_mem_usage = m_total_mem_usage.load ();
+    server_info.mmon_metainfo_mem_usage = m_meta_alloc_count * MMON_METAINFO_SIZE;
+    server_info.num_stat = m_tag_map.size ();
+
+    for (auto it = m_tag_map.begin (); it != m_tag_map.end (); ++it)
+      {
+	server_info.stat_info.emplace_back (it->first, m_stat_map[it->second].load ());
+      }
```

Comment:
Nit. modern c++ way

c++17: more readable and simpler: 
```suggestion
    for (const auto& [stat_name, mem_usage] : m_tag_map)
      {
	server_info.stat_info.emplace_back (stat_name, mem_usage);
      }
```

Reply:
mem_usage have to get from the m_stat_map not m_tag_map.

Reply:
Done.